sdk/core/azure-core/src/main/java/com/azure/core/util/BinaryData.java

```diff
@@ -593,6 +594,21 @@ public InputStream toStream() {
         return new ByteArrayInputStream(this.data);
     }
 
+    /**
+     * Returns a read-only {@link ByteBuffer} representation of this {@link BinaryData}.
+     * <p>
+     * Attempting to mutate the returned {@link ByteBuffer} will throw a {@link ReadOnlyBufferException}.
```

Comment:
One day our lack of proper HTML5 quality JavaDoc will come back to bite us hard. We need to have matching opening / closing elements.

Reply:
The codebase around convinced me this is the way... Is there any checkstyle rule that would enforce this?